environment.yml

```diff
@@ -4,10 +4,10 @@ channels:
   - conda-forge
 
 dependencies:
-  - cftime>=1.3.0  # years<999 get a 0 instead of empty space
+  - cftime=1.2.1  # iris3 needs <=1.2.1; >=1.3.0 - years<999 get a 0 instead of empty space
```

Comment:
Do we need to be this specific? iris has already pinned this like 'cftime<1.3.0', so we probably don't need to.

Reply:
yes! Unfortunately, this is definitely needed because `conda` goes around in order of dependencies and will install `cftime=1.3.1` and `iris=2.4.0` if there is no pin on iris - we now have the pin on `iris` but that will make the environment be solved harder, if at all (I don't know, I haven't tried it yet) but I know the more pins the faster the env solves; we'll remove that once iris will have unpinned it and fix the tests too - a good reminder to fix the `cftime>=1.3.1` tests

Reply:
Having very specific pins like this can make it impossible to solve the environment as dependencies evolve. As you say, the pin is not needed anymore, so i would be happier if it could be removed, here and in other places.

Reply:
hang tight, am tring to see if all works well on me laptop, then commit :+1:

Reply:
ok unpinned it, RE to bear in mind that now we get `mpich=3.4` right away and this one is very problematic, see https://github.com/ESMValGroup/ESMValTool/issues/2006 but doesnt seem to affect Core

Reply:
Then we can pin it in the tool